### DISCUSSION TITLE
Deploying Cadence with Cassandra vs Postgres (recommendation)

### DISCUSSION THREAD

**Discussion by suneeth51** [uber/cadence#4756] [General]
We are planning to deploy Cadence using banzai charts for our production cluster. In our deployment, we have very good support Postgres DB compared to Cassandra. We are looking to deploy Cadence with Postgres. 

Is there any recommendation (Prons/Cons) to use Cassandra v/s Postgres in the Production deployment?

**Comment by demirkayaender** [comment]
We continuously test Postgres and Cassandra with integration tests but internally use Cassandra and a MySQL based proprietary KV store. We don't collect benchmarks for Postgres for now. Functionally it should work but you might want to collect benchmarks by yourself.

**Reply by Instantclustr** [reply]
If you need any assistance in this space reach out to fletcher.kern@instaclustr.com we can assist
